XamarinCommunityToolkit/Behaviors/TextValidationBehavior.shared.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Xamarin.Forms;
+using System.Text.RegularExpressions;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public class TextValidationBehavior : ValidationBehavior
+    {
+        public static readonly BindableProperty MinimumLengthProperty = BindableProperty.Create(nameof(MinimumLength), typeof(int), typeof(TextValidationBehavior), 0, propertyChanged: OnValidationPropertyChanged);
+
+        public static readonly BindableProperty MaximumLengthProperty = BindableProperty.Create(nameof(MaximumLength), typeof(int), typeof(TextValidationBehavior), int.MaxValue, propertyChanged: OnValidationPropertyChanged);
+
+        public static readonly BindableProperty ShouldTrimValueProperty = BindableProperty.Create(nameof(ShouldTrimValue), typeof(bool), typeof(TextValidationBehavior), false, propertyChanged: OnValidationPropertyChanged);
+
+        public static readonly BindableProperty ShouldConvertNullToEmptyValueProperty = BindableProperty.Create(nameof(ShouldConvertNullToEmptyValue), typeof(bool), typeof(TextValidationBehavior), true, propertyChanged: OnValidationPropertyChanged);
```

Comment:
Not sure if we should set the default value to true. This will, by default, mutate the data of the user which can cause effects they aren't aware off. I guess we can leave it for now, just something to think about

Reply:
> Not sure if we should set the default value to true. This will, by default, mutate the data of the user which can cause effects they aren't aware off. I guess we can leave it for now, just something to think about

I believe that's expected behavior -- consider null string as empty. Just for me. Perpahs I am wrong :)